### ISSUE TITLE
missing translation in app accessibility

### ISSUE THREAD

**Issue by cloud2018** [nextcloud/server#15623]
tested with NC 16, Accessibility  1.2.0

The first paragraph in ...nextcloud/settings/user/accessibility is not translated:

![accessibility](https://user-images.githubusercontent.com/34692335/58000397-f563e500-7ad8-11e9-9f69-bf0b199ca67c.png)

I did'nt find the strings in Transifex, so I suspect that they are not translatable in the source

**Comment by wiswedel** [comment]
related? https://github.com/nextcloud/server/issues/15627

**Comment by kesselb** [comment]
Duplicate of #14607
